dask_bigquery/tests/test_core.py

```diff
@@ -387,6 +388,21 @@ def test_arrow_options(table):
     assert ddf.dtypes["name"] == pd.StringDtype(storage="pyarrow")
 
 
+@pytest.mark.parametrize("convert_string", [True, False])
+def test_convert_string(table, convert_string):
```

Comment:
Overall this looks nice. It'd be good to include an `assert_eq` check too to make sure the values are as expected, not just the `name` column dtype.

Reply:
We have this in a few other places already and I can't run tests locally, so don't want to mess around with things too much

Reply:
No problem. Just pushed a small commit to update this (and one other) test.

Reply:
Thx